### DISCUSSION TITLE
Standalone Install Questions

### DISCUSSION THREAD

**Discussion by r1de** [opencardev/crankshaft#482] [Q&A]
Awesome project! Thanks for all the hard work!

Any tips to get a stand alone install on an RPi4 to recognize the phone.
I built all the components myself, aasdk, ilclient, QT 5.1.5 and openauto.
`autoapp` runs, but when I connect my phone, the phone says "Connecting to Car", but I get "Unable to open MTP device xxx,xxx" in OpenAuto. _Something_ is getting detected, but just not correctly...

Of course, everything works when booting to the Crankshaft image.
I do not want to use the image, as it seems limited in what I can do.

Our "Truckputer" needs to do much more than just OpenAuto, it controls my ham radio, has 50GB+ of offline map tiles for Viking, ODBII clients, etc...

Is there some step I am missing to get a stand alone install to recognize my phone?

**Comment by matt2005** [comment]
You need to add udev rules see [Udev rules](https://github.com/opencardev/prebuilts/tree/master/udev)

**Reply by r1de** [reply]
Thanks! Copied that file into place but still getting the error though.
On a whim, I ran the OpenDash install script, the same error pops up, but then it seems to work.
I am still missing some udev magic it seems.
I'll figure it out.
Thanks again!
